share/man/man4/iwn.4

```diff
@@ -130,27 +102,84 @@ For more information on configuring this device, see
 This driver requires the firmware built with the
 .Nm iwnfw
 module to work.
+.Sh HARDWARE
+The
+.Nm
+driver provides support for:
+.Pp
+.Bl -bullet -compact
+.It
+Intel Centrino Advanced-N 6200
+.It
+Intel Centrino Advanced-N 6205
+.It
+Intel Centrino Advanced-N 6230
+.It
+Intel Centrino Advanced-N 6235
+.It
+Intel Centrino Advanced-N + WiMAX 6250
+.It
+Intel Centrino Ultimate-N 6300
+.It
+Intel Centrino Wireless-N 100
+.It
+Intel Centrino Wireless-N 105
+.It
+Intel Centrino Wireless-N 130
+.It
+Intel Centrino Wireless-N 135
+.It
+Intel Centrino Wireless-N 1000
+.It
+Intel Centrino Wireless-N 1030
+.It
+Intel Centrino Wireless-N 2200
+.It
+Intel Centrino Wireless-N 2230
+.It
+Intel Centrino Wireless-N 4965
+.It
+Intel Centrino Wireless-N 5100
+.It
+Intel Centrino Wireless-N 6150
+.It
+Intel Centrino Wireless-N 6200
+.It
+Intel Centrino Wireless-N 6250
+.It
+Intel Centrino Wireless-N + WiMAX 6150
+.It
+Intel Ultimate N WiFi Link 5300
+.It
+Intel Wireless WiFi Link 4965
+.It
+Intel WiFi Link 5100
+.It
+Intel WiMAX/WiFi Link 5150
+.It
+Intel WiMAX/WiFi Link 5350
+.El
 .Sh EXAMPLES
 Join an existing BSS network (i.e., connect to an access point):
 .Bd -literal -offset indent
-ifconfig wlan create wlandev iwn0 inet 192.168.0.20 \e
+# ifconfig wlan create wlandev iwn0 inet 192.168.0.20 \e
```

Comment:
This should really become an TEST-NET prefix[1] but that's probably a totally different pass over man pages.

[1] https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml

Reply:
That would be so sharp! Can we use cidr and put it on one line?